Hindi_transcripts/school/Epic Quest/sharedAssets/C1_INTRO_EQ_English.txt

```diff
@@ -0,0 +1,3 @@
+<break time="1900ms"/><prosody rate="1.2">नमस्कार</prosody> <break time="800ms"/><prosody rate="1.3"> आपने अपने टैबलेट का उपयोग करके कितना सीखा हैं? इस मज़ेदार यात्रा में दरवाज़ों को पार करने के लिए अपने नए कौशल का उपयोग करें.</prosody> <break time="1000ms"/> हम इसे  Epic quest कहते हैं.<break time="300ms"/> <prosody rate="1.2">सड़क के अंत में एक गाँव होगा जिसे आप देख सकते हैं और यात्रा कर सकते हैं. सुराग ढूंढो और रास्ते में वस्तुओं और जानवरों को इकट्ठा करते रहो</prosody> <break time="2000ms"/><prosody rate="1.2"> आप उन वस्तुओं को जोड़ने और गांव में रहस्यों को सुलझाने के लिए सुराग का उपयोग करने में सक्षम होंगे</prosody> <break time="7500ms"/><prosody rate="1.2"> सड़क यात्रा करते रहें, अधिक सुराग खोजें  और अक्सर गांव का दौरा करते रहें</prosody><break time="600ms"/> Good luck  <break time="1000ms"/>
```

Comment:
आप उन वस्तुओं को जोड़ने और गांव में रहस्यों को सुलझाने के लिए सुराग का उपयोग करने में सक्षम होंगे

Please confirm this with @agarg8487 once

Reply:
सड़क यात्रा करते रहें, अधिक सुराग खोजें  और अक्सर गांव का दौरा करते रहें

Even this does not sound correct.